_drafts/2017-09-01-first-review-experiences.md

```diff
@@ -0,0 +1,137 @@
+---
+name: first-review-experiences
+layout: post_discourse
+title: Experiences as a first time rOpenSci package reviewer
+date: 2017-09-01
+authors:
+  - name: Verena Haunschmid
+    url: http://blog.haunschmid.name
+tags:
+- R
+- software
+- review
+- ropensci
+- onboarding
+---
+
+It all started January 26^th^ this year when I signed up to volunteer as
+a reviewer for R packages submitted to rOpenSci. My main motivation for
+wanting to do package reviews was to learn something new and to
+contribute to the R open source community.
+
+Three months later I was contacted by [Ma�lle Salmon](https://twitter.com/ma\_salmon) asking whether I was interested in
+reviewing the R package ['patentsview'](https://github.com/ropensci/patentsview) for rOpenSci. And yes, I
+was! To be honest I was a little bit thrilled.
+
+The packages are submitted for review to rOpenSci via an issue to their
+github repository and also the reviews happen there. So you can check out all
+previous package submissions and reviews. With all the information you
+get from rOpenSci and also the help from the editor it is straight
+forward to do the package review. Before I started my review I read the
+reviewer guides (links below) and checked out a few of the existing
+reviews. I installed the package 'patentsview' from github and also
+downloaded the source code so I could check out how it was implemented.
+There are several R packages that helped me get started with my review,
+e.g. ['devtools'](https://github.com/hadley/devtools) and
+['goodpractice'](https://github.com/MangoTheCat/goodpractice). These
+packages can also help you when you start writing your own packages. An
+example for a very useful method is 'devtools::spell\_check()', which
+performs a spell check on the package description and on manual pages.
+At the beginning I had an issue with 'goodpractice::gp()' but Ma�lle Salmon 
+(the editor) helped me resolve it. After that I started my review
```

Comment:
I really like this workflow description! So much so that I think you could start the paragraph with "I started my review...", and in a second paragraph in this section talk abut the packages you used to help.